cpp/tests/utilities/column_utilities.cu

```diff
@@ -69,5 +70,9 @@ void expect_equal_buffers(void const* lhs, void const* rhs,
                             typed_rhs));
 }
 
+detail::column_wrapper::operator column_device_view() const {
+  return *column_device_view::create(wrapped->view());
```

Comment:
This is an error. `column_device_view`s cannot be created with a conversion operator (there's a reason `cudf::column` doesn't have one).

`column_device_view::create` returns a `unique_ptr<column_device_view>`.

What's happening here is effectively:
```
std::unique_ptr<column_device_view, deleter> view_ptr = column_device_view::create(wrapped->view());
column_device_view bad_reference = *view_ptr;
return bad_reference; 
// `view_ptr` goes out of scope and is destroyed. `bad_reference` refers to a delete object!!!
``` 

Instead in tests you should just do:
```
column_wrapper w(...);
auto device_view = column_device_view::create(w); // implicit conversion of column_wrapper to column_view
// use device_view
```

Reply:
```
column_device_view bad_reference = *view_ptr;
```
here bad_reference is not a reference variable. so it should be its own object with contents copied from `*view_ptr`?

Reply:
removed.